src/App.jsx

```diff
@@ -0,0 +1,28 @@
+import Navbar from './components/navbar'
+import About from './components/about'
+import Home from './components/home'
+import Upload from './components/upload';
+import {
+  BrowserRouter as Router,
+  Routes,
+  Route,
+  Link
+} from "react-router-dom";
+
+
+function App() {
+  return (
+    <Router>
+      <div className="">
+        <Navbar />
+        <Routes>
+          <Route exact path="/home" element={<><Home /></>}></Route>
```

Comment:
you can remove these extra <> </>

Reply:
Everything else looks great @AdityaShekharTiwary  , can you add your review too? @Supriya-roy9  we can merge it then